R/expand.R

```diff
@@ -15,6 +15,7 @@
 fct_expand <- function(f, ..., after = Inf) {
   f <- check_factor(f)
   check_dots_unnamed()
+  check_number_decimal(after)
```

Comment:
Minimum value of `0` i believe?

Also is it `check_number_whole()`?

Reply:
`check_number_whole(Inf)` throws an error 😞